Core/include/Acts/Propagator/DefaultExtension.hpp

```diff
@@ -1,18 +1,245 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2018-2019 CERN for the benefit of the Acts project
+// Copyright (C) 2018-2023 CERN for the benefit of the Acts project
 //
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 #pragma once
 
-#include "Acts/Propagator/detail/GenericDefaultExtension.hpp"
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/Definitions/TrackParametrization.hpp"
+#include "Acts/Utilities/VectorHelpers.hpp"
+
+#include <array>
 
 namespace Acts {
 
-/// @brief A typedef for the default GenericDefaultExtension with double.
-using DefaultExtension = detail::GenericDefaultExtension<double>;
+/// @brief Default evaluater of the k_i's and elements of the transport matrix
+/// D of the RKN4 stepping. This is a pure implementation by textbook.
+struct DefaultExtension {
```

Comment:
This just *inlines* the `GenericDefaultExtension` with `double` as template argument, yes?

Reply:
yes exactly